app/components/emoji_picker/emoji_picker.js

```diff
@@ -214,7 +229,7 @@ class EmojiPicker extends PureComponent {
                 <View style={styles.container}>
                     <SectionList
                         showsVerticalScrollIndicator={false}
-                        style={styles.listView}
+                        style={[styles.listView, {width: this.state.deviceWidth}]}
```

Comment:
Instead of explicitly passing in the width here, couldn't we just set margins or padding on the container?